package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Common Creative search",
   "author": "sebworks",
   "private": true,
-  "type": "module",
```

Comment:
I think that this should probably be a separate issue. It is not strictly within the scope of a fix for pinterest sharing.

Please revert this change, and open a separate issue for it.